streaming/src/main/scala/org/apache/carbondata/streaming/StreamHandoffRDD.scala

```diff
@@ -299,8 +302,8 @@ object StreamHandoffRDD {
   def executeStreamingHandoff(
       carbonLoadModel: CarbonLoadModel,
       sparkSession: SparkSession,
-      handoffSegmenId: String
-  ): Unit = {
+      operationContext: OperationContext,
```

Comment:
why operationContext is needed

Reply:
operation context has CarbonLoadDataCommands which are needed in PreAggregateListeners.